redhat-cop/automate_tower_genie_collections
# Remove role defaults to enable role conditions

### What does this PR do?
Currently (As mentioned in #68) many roles have defaults set. This means we cannot do logic to include the roles as by adding `- {role: organizations, when: tower_organizations is defined, tags: organizations}`, `tower_organizations` will always be set as defaulted in the role.

It doesn't actually make sense to have this as a default anyway (in my opinion) so this PR removes defaults.

All of the roles (except 2, there will be another PR for that) have examples of how to run the role, and the sensible first point of call for finding out how to run is probably the documentation (the README) rather than the defaults.

### How should this be tested?
The automation *should* still work, and to test this specifically you can remove the pre-tasks section of `playbooks/configure_tower.yml` and observe that all roles are skipped.

### Is there a relevant Issue open for this?
resolves #68 
resolves #69 
resolves #70 
resolves #71 

### Other Relevant info, PRs, etc.
N/A
